modules/build/src/main/scala/scala/build/Build.scala

```diff
@@ -255,13 +259,29 @@ object Build {
         (mainBuild, extraBuilds)
       }
 
+    def doPostProcess(build: Seq[Build], scope: Scope) = build.collect {
```

Comment:
Should we ignore errors from post process?

Also a small nit, you can combine `collect` and `map` into one call